pages/content/amp-dev/documentation/guides-and-tutorials/contribute/contribute-documentation/documentation-types.md

```diff
@@ -0,0 +1,274 @@
+---
+$title: Documentation types 
+$order: 1
+description: 'Types of documentaion contributions accepted on amp.dev'
+formats:
+  - websites
+  - stories
+  - ads
+  - email
+author: CrystalOnScript
+---
+Below is a brief outline of the types of documentation contributions accepted on amp.dev. 
```

Comment:
Suggestion: add a TOC listing all available contribution types

Reply:
Do you think we need an additional TOC that different from the generated one? The headlines are just the different contribution types. 
![image](https://user-images.githubusercontent.com/23297417/62481083-4abda300-b766-11e9-93b7-75ba99e94a9a.png)

Reply:
lol - no

Reply:
Although, it would work better inline.

Reply:
Ah, fair point! I'll update it